contexts/zcap-v1.jsonld

```diff
@@ -53,6 +53,11 @@
       "@id": "https://w3id.org/security#parentCapability",
       "@type": "@id"
     },
+    "proof": {
+      "@id": "https://w3id.org/security#proof",
+      "@type": "@id",
+      "@container": "@graph"
+    },
     "referenceId": "https://w3id.org/security#referenceId"
   }
 }
```

Comment:
Looks like this needs the usual newline.

Reply:
Added newline & changelog entry, thanks.